docs/index.rst

```diff
@@ -268,86 +206,76 @@ Topic Management
+4. View the existing KSQL streams. (If you are using the KSQL CLI, at the ``ksql>`` prompt type ``SHOW STREAMS;``).
 
-     .. figure:: images/ksql_query_topic.png
-        :alt: image
+   .. figure:: images/ksql_streams_list.png
+      :alt: image
 
-7. **DEVELOPMENT -> KSQL -> STREAMS**: Create a new stream from an existing topic. Click on the button ``Create Stream`` and follow the prompts.
+5. Describe the schema (fields or columns) and source and sink of an existing KSQL stream. Click on ``WIKIPEDIA``.
 
-8. **DEVELOPMENT -> KSQL -> QUERY EDITOR**: View the configured KSQL server properties set in the docker-compose.yml file. In the query editor, type ``SHOW PROPERTIES;`` and then click on the ``Run query`` button.
+   .. figure:: images/wikipedia_describe.png
+      :alt: image
 
-     .. figure:: images/ksql_properties.png
-        :alt: image
+6. View the existing KSQL tables. (If you are using the KSQL CLI, at the ``ksql>`` prompt type ``SHOW TABLES;``).
 
-9. In this demo, KSQL is run with Confluent Monitoring Interceptors configured which enables |c3| Data Streams to monitor KSQL queries. The consumer group names ``ksql_query_`` correlate to the KSQL query names above, and |c3| is showing the records that are incoming to each query.
+   .. figure:: images/ksql_tables_list.png
+      :alt: image
 
-* View throughput and latency of the incoming records for the persistent KSQL "Create Stream As Select" query ``CSAS_WIKIPEDIABOT``, which is displayed as ``ksql_query_CSAS_WIKIPEDIABOT`` in |c3|.
+7. View the existing KSQL queries, which are continuously running. (If you are using the KSQL CLI, at the ``ksql>`` prompt type ``SHOW QUERIES;``).
 
-   .. figure:: images/ksql_query_CSAS_WIKIPEDIABOT.png
+   .. figure:: images/ksql_queries_list.png
       :alt: image
 
-* View throughput and latency of the incoming records for the persistent KSQL "Create Table As Select" query ``CTAS_EN_WIKIPEDIA_GT_1``, which is displayed as ``ksql_query_CTAS_EN_WIKIPEDIA_GT_1`` in |c3|.
+8. View messages from different KSQL streams and tables. Right click on your stream of choice, select ``Query`` which takes you to the Query Editor with a pre-populated query such as ``select * from WIKIPEDIA;`` and it will show results for newly arriving data.
 
-   .. figure:: images/ksql_query_CTAS_EN_WIKIPEDIA_GT_1.png
+   .. figure:: images/ksql_query_topic.png
       :alt: image
 
-* View throughput and latency of the incoming records for the persistent KSQL "Create Stream As Select" query ``CTAS_EN_WIKIPEDIA_GT_1_COUNTS``, which is displayed as ``ksql_query_CSAS_EN_WIKIPEDIA_GT_1_COUNTS`` in |c3|.
+9. View the configured KSQL server properties set in the docker-compose.yml file. In the query editor, type ``SHOW PROPERTIES;`` and then click on the ``Run query`` button.
```

Comment:
I stumbled here slightly as reading the text as we've put the user inside the Editor query view for the `WIKIPEDIA` stream.   So to make the work you have to click a few buttons to get back to the "main" KSQL editor view.   Something like this might be more explicit?
```suggestion
9. From the main `KSQL Editor` view, you can see the configured KSQL server properties by entering the query, ``SHOW PROPERTIES;`` and then clicking the ``Run`` button.   You can cross check these properties with the `docker-compose.yml` file.
```

Reply:
Suggest:

9. Click **KSQL Editor** and run the ``SHOW PROPERTIES;`` statement. You can see the configured KSQL server properties and check these values with the ``docker-compose.yml`` file.

Reply:
Ack